sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1625,6 +1625,17 @@ object SQLConf {
     .intConf
     .createWithDefault(25)
 
+  val MAX_PLAN_STRING_LENGTH = buildConf("spark.sql.maxPlanLength")
```

Comment:
"maxPlanStringLength` would be a better name.

I'm a little unsure whether it should be in `SQLConf` or `StaticSQLConf`, but leaving it here makes it more flexible, so seems fine to me.